src/components/Card/TimeSelectorCard.tsx

```diff
@@ -0,0 +1,50 @@
+import { FC, useState } from 'react'
+import { ITimestamp } from '../time-selector/TimeSelectorInterfaces'
+import moment from 'moment'
+import './Card.css'
+
+interface ICardProps {
+    name: string
+}
+
+export const TimeSelectorCard: FC<ICardProps> = ({ name }) => {
+
+    const [startTime, setStartTime] = useState<string>('')
+    const [endTime, setEndTime] = useState<string>('')
+    const [timestamp, setTimestamp] = useState<ITimestamp[]>([])
+
+    const getTimestamps = () => {
+        return ( 
+            timestamp?.map((time, index) => {
+                const { startTime, endTime } = time
+                return <div key={`${index}${startTime}${endTime}`}>
```

Comment:
I think this key value could overlap with other dropdown's key value. Please use a unique key value for each drop down.